packages/block-editor/src/components/inner-blocks/README.md

```diff
@@ -113,10 +113,23 @@ If locking is not set in an `InnerBlocks` area: the locking of the parent `Inner
 
 If the block is a top level block: the locking of the Custom Post Type is used.
 
-### `placeholderType`
-* **Type:** `String`
-* **Default:** - uses the `DefaultBlockAppender` (`packages/block-editor/src/components/default-block-appender/index.js`) to automatically insert a `paragraph` Block. 
+### `renderAppender`
+* **Type:** `String|Function`
+* **Default:** - `null`. By default the `auto-insert` behaviour (see below) is utilised.
 * **Options:** 
-    - `block-appender` - forces the use of the Block List Appender (`packages/block-editor/src/components/block-list-appender/index.js`) to display a "Add Block" message with an icon which fills the Block preview. No default Block is inserted.
+    - `block-appender` - uses the Block List Appender (`packages/block-editor/src/components/block-list-appender/index.js`) to display a "Add Block" message with an icon which fills the Block preview. No default Block is inserted.
```

Comment:
is `-appender` useful here? seems a bit redundant

Reply:
I guess it's not any more now we have `appender` in the prop name.

Reply:
Fixed